.travis.yml

```
@@ -22,10 +22,10 @@ jobs:
     - name: "Rust: stable - Examples"
       rust: stable
       script:
-        # NOTE: We pipe `tail -f /dev/null` into `yarn start`, which internally
-        #   calls `truffle develop`, so that the process' STDIN stays open, as
-        #   `truffle develop` exits as soon as it gets an EOF from STDIN.
-        - tail -f /dev/null | yarn --cwd examples/truffle start > /dev/null &
+        - npm install -g truffle
+        # NOTE: We pipe `tail -f /dev/null`, so that the process' STDIN stays
+        # open, as `truffle develop` exits as soon as it gets an EOF from STDIN.
+        - (cd examples/truffle; tail -f /dev/null | truffle develop > /dev/null) &
```

What if you use `npx truffle` here? I'm afraid this build is quite brittle as it newer versions of truffle can break it un unexpected ways.
```suggestion
        - (cd examples/truffle; tail -f /dev/null | npx truffle develop > /dev/null) &
```

---

If that doesn't work, you can
- Update to latest version of Truffle (the Ubuntu bionic thing might be why #392 is failing?)
- Even run `truffle` directly via `./node_modules/.bin/truffle`.

---

`./node_modules/.bin/truffle` does not work (it seems to hang forever). i'm guessing `yarn start` and `npx truffle` call that same binary which has some issue. With `strace` I see that it seems to be waiting on the unix socket `/tmp/truffle.develop` but I don't know what that means.
I am going to merge this PR now so that CI is at least fixed now even though it could be brittle in the future. Will make issue about this though.